docker-bake.hcl

```diff
@@ -132,6 +135,32 @@ target "ld64-tgz" {
         "linux/arm/v7"
     ]
     cache-to = ["type=inline"]
+    output = ["./ld64-tgz"]
+}
+
+target "ld64-linux-amd64-tgz" {
```

Comment:
Why do we need these separate targets in here. `ld64-tgz` already builds all combinations.

Reply:
It's to reduce build team so we can distribute builds across multi runners in the pipeline, see: https://github.com/crazy-max/xx/actions/runs/4155296585